components/core/tools/scripts/lib_install/check-cmake-version.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Exit on any error
+set -e
+
+# Error on undefined variable
+set -u
+
+# Get the installed cmake version string
+cmake_version=$(cmake -E capabilities | jq --raw-output ".version.string")
+
+# Check if version is 4.0 or higher
+# shellcheck disable=SC2071
+if ! [[ $cmake_version < "4" ]]; then
```

Comment:
Just curious, is the quote around `4` to make the string comparison explicit?
Also, this will only work if the reported cmake version is less than 10, right?

Reply:
@Bill-hbrhbr Let's switch back to your original solution.